packages/console/src/pages/Connectors/index.tsx

```diff
@@ -152,12 +150,6 @@ function Connectors() {
             colSpan: 5,
             render: (connectorGroup) => <ConnectorTypeColumn connectorGroup={connectorGroup} />,
           },
-          {
```

Comment:
Should also adjust the colspan of the rest of the columns. Title : rest ratio should be 6:10

Reply:
Title:Type:Delete ratio should be 6:9:1.